rollup.config.js

```diff
@@ -1,42 +1,30 @@
-const nodeResolve = require('@rollup/plugin-node-resolve')
-const resolve = ("function" === typeof nodeResolve )?nodeResolve:nodeResolve.nodeResolve;
 const commonjs = require('@rollup/plugin-commonjs')
-//const closure = require('rollup-plugin-closure-compiler' );
-const terser = require('rollup-plugin-terser').terser
+const { append } = require('rollup-plugin-insert')
+const { terser } = require('rollup-plugin-terser')
 const pkg = require('./package.json')
 
+/**
+ * @type {import('rollup').RollupOptions[]}
+ */
 module.exports = [
-    // ES6 Modules Non-minified
+    // CommonJS Non-minified
     {
-        input: 'lib/index.js',
+        input: pkg.module,
         output: {
-            file: pkg.browser.replace(/\.js$/, '.mjs'),
```

Comment:
The `browser` field was `lib/jsox/es6.js.gz`, I don't think it's working correctly.